core/drivers/crypto/hisilicon/include/hpre_ecc.h

```diff
@@ -0,0 +1,113 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, HiSilicon Technologies Co., Ltd.
+ */
+
+#ifndef __HPRE_ECC_H__
+#define __HPRE_ECC_H__
+
+#define ECC_DH_IN_PARAM_NUM	2
```

Comment:
Do you expect these to be used outside `hpre_ecc.c` or could they be moved into that C file?